puzzlesolver/puzzles/_models/puzzle.py

```diff
@@ -167,6 +167,26 @@ def generateSolutions(self):
         """
         return []
 
+    #################################################################
+    # Player methods
+    #################################################################
+
+    def playPuzzle(self, moves):

```

Comment:
Why does the playPuzzles have a move argument instead of just using generateMoves?

Reply:
After merging, I'll fix the functionality for this.